Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -36,7 +36,7 @@ internal enum KeepAliveState {
 
 /// A representation of the request line and header fields of a HTTP request.
 public struct HTTPRequestHead: Equatable {
-    private final class _Storage {
+    fileprivate final class _Storage {
```

Comment:
I change the access level from `private` to `fileprivate` to be able to mark `_Storage` as non-sendable which is only possible in an extension. No strong optioned on that though, so happy to make it `private` again.

Reply:
I think we don't need to mark internal types as non-sendable, so I'm happy for us to throw this away.

Reply:
`private` again https://github.com/apple/swift-nio/pull/2213/commits/9d78041cee755bf0870aac569bb583545d0e8321